static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,113 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
```

Comment:
let's remove this remark about `dvc run` and this sentence. even data registries might contain "stages" in certain case, and I think it's not that important to complicate it here.

Reply:
Removing...

Reply:
I mean, do we really want to keep the part about `dvc add`? We don't care that much about the way artifacts are generated, right?

Reply:
The **Example** uses `dvc add` and most data registries I think would be built mostly with `dvc add`, as it's unlikely you'll want stages in a data registry, no? These are assumptions and people always break expectations but would it technically still be called a "data registry" if it has a bunch of pipelines? I'm not sure.

I think this note isn't restrictive though, it's just a brief comment that may give the user a better idea of that a data registry is if they're already familiar with `dvc add`.

Reply:
p.s. "Such a project would not have stages..." is gone.